src/test/java/com/microsoft/lst_bench/common/LSTBenchmarkExecutorTest.java

```diff
@@ -51,6 +58,11 @@ void tearDown() {
     if (telemetryDbFile.exists()) {
       telemetryDbFile.delete();
     }
+
+    File telemetryDbWalFile = new File(telemetryDbFileName.toString() + ".wal");
```

Comment:
I am a bit puzzled too about these files. It seems duckdb creates the write-ahead-log files by default. I did not explore all the configs to disable it. Try delete this line and running the test, you will find `.wal` files in the working directory. I specifically added this code as a "safe" cleanup operation after the test completes so that such files do not accumulate over time.